zuul.d/edpm_multinode.yaml

```diff
@@ -190,6 +190,9 @@
 - job:
     name: podified-multinode-hci-deployment-crc
     parent: podified-multinode-hci-deployment-crc-3comp
+    vars:
+      cifmw_extras:
```

Comment:
You probably need to also add `- '@scenarios/centos-9/multinode-ci.yml'` here as first element of this list. Since zuul will not merge with the already defined cifmw_extras in: https://github.com/openstack-k8s-operators/ci-framework/pull/1347/files#diff-907d84a405081e9db4210be5cf0000e7c775eec6e874234f282ad829317e6975R174

Reply:
Done